app/controllers/api/api_controller.rb

```diff
@@ -2,13 +2,14 @@ module Api
   class ApiController < Api::BaseController
     def index
       res = {
-        :name        => Settings.base.name,
-        :description => Settings.base.description,
-        :version     => Settings.base.version,
-        :versions    => entrypoint_versions,
-        :settings    => user_settings,
-        :identity    => auth_identity,
-        :server_info => server_info,
+        :name                => Settings.base.name,
+        :description         => Settings.base.description,
+        :version             => Settings.base.version,
+        :versions            => entrypoint_versions,
+        :settings            => user_settings,
+        :identity            => auth_identity,
+        :server_info         => server_info,
+        :productization_info => productization_info
```

Comment:
I personally prefer `productization_info` because it is very relevant to `productization`